uploader.json

```diff
@@ -3078,6 +3078,11 @@
             ],
             "symbolic_name": "virtlogd_conf"
         },
+        {
+            "file": "/sys/kernel/mm/swap/vma_ra_enabled",
+            "pattern": [],
+            "symbolic_name": "vma_ra_enabled_s390x"
```

Comment:
@xhe123 the symbolic name needs to match the spec name in insights core, `vma_ra_enabled`.